opentelemetry-contrib/src/datadog/mod.rs

```diff
@@ -200,13 +195,15 @@ impl trace::SpanExporter for DatadogExporter {
             Err(_) => return trace::ExportResult::FailedNotRetryable,
         };
 
-        let resp = self
-            .client
-            .post(self.request_url.clone())
-            .header(CONTENT_TYPE, self.version.content_type())
+        let req = match Request::builder()
```

Comment:
I haven't tried it, yet, but I feel like this is missing the method and url. Should this be something like this?
```rust
let req = match Request::post(self.request_url.clone())
    .header("content-type", self.version.content_type())
    .body(data)
```

Reply:
Good catch! I guess I forget it somehow. Fixed.